app/views/candidate_mailer/ucas_match_initial_email_duplicate_applications.text.erb

```diff
@@ -0,0 +1,22 @@
+Dear <%= @application_form.full_name %>
+
+It looks like you submitted an application for <%= @course_name_and_code %> at <%= @provider_name %> on both DfE Apply for teacher training and UCAS Teacher Training Apply.
+
+This has been flagged to us, as well as to <%= @provider_name %>, as applying for the same course on both services is not allowed, as outlined by the terms of use.
+
+Action required – withdraw your duplicate application
```

Comment:
Does this need to be bold?

Reply:
All existing emails are in plain text format, so I followed the same pattern. There is also no formatting included in the email templates.

Reply:
There is: see https://github.com/DFE-Digital/apply-for-teacher-training/blob/master/app/views/candidate_mailer/deferred_offer.text.erb which renders as https://qa.apply-for-teacher-training.service.gov.uk/rails/mailers/candidate_mailer/deferred_offer

Reply:
I was looking for html emails 🙄